### PR TITLE
Launches a ubuntu instance with multipass

### PR DESCRIPTION
**Pull request by Sanlung**

### CODE REVIEW

File: webserver.sh [hunk 1]
```diff
@@ -12,7 +12,8 @@ else
     if ( brew --version )
     then
       echo "brew already installed"
-    else 
+    else
+	  echo "brew not installed ... installing brew"
```
Comment (brianrichiesr) [inline webserver.sh]: Indentation needs fixing

**Review by brianrichiesr — CHANGES_REQUESTED** [review]

File: webserver.sh (hunk 1 above)
Comment (rwc) [reply webserver.sh]: Fixed in latest commit.

**Review by brianrichiesr — APPROVED** [review]

**Review by rwc — APPROVED** [review]
Ship

**Comment by brianrichiesr** [comment]
Ship It

**Pull request merged**